CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+# v24.6.2
+- [Icons] Add IvermectinGlyph([#1209](https://github.com/curology/radiance-ui/pull/1209))
+- [Icons] Add KetoconazoleGlyph([#1209](https://github.com/curology/radiance-ui/pull/1209))
```

Comment:
I'm not sure that this matters _too_ much, but the publishing instructions recommend [updating the changelog as part of the versioning](https://www.notion.so/curology/Radiance-UI-3e2736219ba14a91b14075a3d9cd922e#dbc69b79e3a8474aa6da25f9dd7a781a) step and putting it in that commit.